test/unit/event/owner/hcs-did-create-did-owner-event.test.ts

```diff
@@ -0,0 +1,144 @@
+import { PrivateKey } from "@hashgraph/sdk";
+import { DidError, Hashing, HcsDidCreateDidOwnerEvent, HcsDidEventTargetName } from "../../../../dist";
+
+describe("HcsDidCreateDidOwnerEvent", () => {
+    const privateKey = PrivateKey.fromString(
+        "302e020100300506032b6570042204209044d8f201e4b0aa7ba8ed577b0334b8cb6e38aad6c596171b5b1246737f5079"
+    );
+    const identifier = `did:hedera:testnet:${Hashing.multibase.encode(privateKey.publicKey.toBytes())}_0.0.29613327`;
```

Comment:
This looks like a hard coded account id, shouldn't it be something that we pull from a .env/secret for testing ?

Reply:
The number is not a secret, and has to be fixed to be able to check the outputs. 

Could have been _0.0.12345678 to be clearer it is a place holder; but i really do not want to prepare a new set of test data ;) 

FTR. the id is actually a topic_id, not an account_id.